docs/EXTENDED-ARCHIVES.md

```diff
@@ -112,15 +112,17 @@ the image.
 Here's an example of a real-world manifest for Raspbian Jessie
 
 ```json
-{
-  "publisher": {
-    "name": "Raspberry Pi",
-    "url": "https://www.raspberrypi.org",
-    "logo": "raspberrypi.svg",
-    "colorScheme": {
-      "background": "#535760",
-      "text": "#FFFFFF",
-      "primary": "#5793db"
+[
+  {
+    "publisher": {
+      "name": "Raspberry Pi",
+      "url": "https://www.raspberrypi.org",
+      "logo": "raspberrypi.svg",
+      "colorScheme": {
+        "background": "#535760",
+        "text": "#FFFFFF",
+        "primary": "#5793db"
+      }
```

Comment:
BTW what's the use-case for having multiple publishers in a single extended archive?